### PR TITLE
Service Definition API doc

### PR DESCRIPTION
**Pull request by kayayarai**
### What does this PR do?
Publishes beta API docs for service definition endpoints

### Motivation
docs-3216

### Preview
https://docs-staging.datadoghq.com/kari/docs-3216-svc-catalog-api/tracing/faq/service_definition_api/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/faq/service_definition_api.md [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+### Request
+
+#### Body data (required)
+
+| Field                       | Type            | Description |
+| --------------------------- | --------------- | ------------------------------------------------------- |
+| schema-version&nbsp;[_required_] | string          | Version of the service definition schema being used. Only value `v2` is supported.|
+| dd-service&nbsp;[_required_]     | string          | Unique identifier of the service. Must be unique across all services and it used to match with a service in Datadog. |
+| team                        | string          | Name of the team responsible for the service. |
+| contacts                    | object          | List of contacts on the team. |
```
Comment (hyperloglogy) [inline content/en/tracing/faq/service_definition_api.md:35]: `contacts` is optional

File: content/en/tracing/faq/service_definition_api.md [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;type&nbsp;[_required_]      | string          | Contact type  |
+| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;name                        | string          | Contact name  |
+| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;contact&nbsp;[_required_]   | string          | Contact value |
+| links                       | object          | List of important links related to the service (for example, a runbook). |
```
Comment (hyperloglogy) [inline content/en/tracing/faq/service_definition_api.md:39]: `links` is optional

Comment (hyperloglogy) [reply content/en/tracing/faq/service_definition_api.md:39]: these top level objects are optional: 
* contacts
* links
* docs
* repos
* extensions
* tags
* integrations

File: content/en/tracing/faq/service_definition_api.md [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+```
+
+### Arguments
+
+#### Header parameters
+
+| Required field  | Description |
+| ---------- | ----------- |
+| `DD-API-KEY` | Identifies an organization. To create or reuse existing keys, go to the [API keys page][2]. |
+| `DD-APPLICATION-KEY` | Identifies a user. To create or reuse existing keys, to to the [Application keys page][3]. |
```
Comment (jtappa) [inline content/en/tracing/faq/service_definition_api.md:233]:
```suggestion
| `DD-APPLICATION-KEY` | Identifies a user. To create or reuse existing keys, go to the [Application keys page][3]. |
```

File: content/en/tracing/faq/service_definition_api.md [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+
+| Required field  | Description |
+| ---------- | ----------- |
+| `service_name` | Identifies the service to delete its definition. |
+
+#### Header parameters
+
+| Required field  | Description |
+| ---------- | ----------- |
+| `DD-API-KEY` | Identifies an org. To create or reuse existing keys, go to the [API keys page][2]. |
```
Comment (jtappa) [inline content/en/tracing/faq/service_definition_api.md:263]:
```suggestion
| `DD-API-KEY` | Identifies an organization. To create or reuse existing keys, go to the [API keys page][2]. |
```

File: content/en/tracing/faq/service_definition_api.md [hunk 5]
```diff
@@ -0,0 +1,280 @@
...
+| `DD-APPLICATION-KEY` | Identifies a user. To create or reuse existing keys, to to the [Application keys page][3]. |
```
Comment (jtappa) [inline content/en/tracing/faq/service_definition_api.md:264]:
```suggestion
| `DD-APPLICATION-KEY` | Identifies a user. To create or reuse existing keys, go to the [Application keys page][3]. |
```

File: content/en/tracing/faq/service_definition_api.md [hunk 6]
```diff
@@ -0,0 +1,280 @@
...
+```
+
+### Arguments
+
+#### Header parameters
+
+| Required field  | Description |
+| ---------- | ----------- |
+| `DD-API-KEY` | Identifies an organization. To create or reuse existing keys, go to the [API keys page][2]. |
+| `DD-APPLICATION-KEY` | Identifies a user. To create or reuse existing keys, to to the [Application keys page][3]. |
```
Comment (jtappa) [inline content/en/tracing/faq/service_definition_api.md:24]:
```suggestion
| `DD-APPLICATION-KEY` | Identifies a user. To create or reuse existing keys, go to the [Application keys page][3]. |
```

File: content/en/tracing/faq/service_definition_api.md [hunk 7]
```diff
@@ -0,0 +1,280 @@
...
+
+### Request
+
+#### Body data (required)
+
+| Field                       | Type            | Description |
+| --------------------------- | --------------- | ------------------------------------------------------- |
+| schema-version&nbsp;[_required_] | string          | Version of the service definition schema being used. Only value `v2` is supported.|
+| dd-service&nbsp;[_required_]     | string          | Unique identifier of the service. Must be unique across all services and it used to match with a service in Datadog. |
```
Comment (jtappa) [inline content/en/tracing/faq/service_definition_api.md:33]:
```suggestion
| dd-service&nbsp;[_required_]     | string          | Unique identifier of the service. Must be unique across all services, and used to match with a service in Datadog. |
```

**Review by jtappa — CHANGES_REQUESTED** [review]

**Review by jtappa — APPROVED** [review]

**Pull request merged**